scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -447,7 +447,7 @@ <h6>Phenotype terms (HPO):</h6>
 {% endmacro %}
 
 {% macro causative_button(variant, case, institute) %}
-  {% if causatives in case and variant._id in case.causatives %}
+  {% if case.causatives and variant._id in case.causatives %}
```

Comment:
Ah, but what about when it doesn't exist in the first place? Dont you have to:

`{% if causatives in case and case.causatives and variant._id in case.causatives %}`

Reply:
Apparently not. It should be somewhere in the documentation

Reply:
Ah. No. 
```suggestion
  {% if 'causatives' in case and case.causatives and variant._id in case.causatives %}
```

Reply:
Tested and it works?